.buildkite/ftr_configs.yml

```diff
@@ -87,12 +87,10 @@ disabled:
   - x-pack/test_serverless/api_integration/test_suites/observability/config.ts
   - x-pack/test_serverless/api_integration/test_suites/search/config.ts
   - x-pack/test_serverless/api_integration/test_suites/security/config.ts
-  - x-pack/test_serverless/api_integration/test_suites/reporting/config.ts
   - x-pack/test_serverless/functional/test_suites/common/config.ts
   - x-pack/test_serverless/functional/test_suites/observability/config.ts
   - x-pack/test_serverless/functional/test_suites/search/config.ts
   - x-pack/test_serverless/functional/test_suites/security/config.ts
-  - x-pack/test_serverless/functional/test_suites/reporting/config.ts
```

Comment:
By design, the only serverless test suites are: `observability`, `search`, `security` and `common`. Reporting tests should go under `common`.